src/components/backend-ai-agent-summary-list.ts

```diff
@@ -0,0 +1,501 @@
+          --component-min-width: 400px;
+        }
+
+        backend-ai-dialog {
+          --component-width: 350px;
+        }
+
+        img.indicator-icon {
+          width: 16px !important;
+          height: 16px !important;
+        }
+
+
+        lablup-progress-bar {
+          --progress-bar-width: 70px;
+          border-radius: 3px;
+          height: 10px;
+          --mdc-theme-primary: #3677eb;
+          --mdc-linear-progress-buffer-color: #98be5a;
+          margin-bottom: 0;
+        }
+
+        lablup-progress-bar.cpu {
+          --progress-bar-height: 7px;
+        }
+
+        lablup-progress-bar.cuda {
+          --progress-bar-width: 80px;
+          --progress-bar-height: 15px;
+          margin-bottom: 5px;
+        }
+
+        lablup-progress-bar.mem {
+          --progress-bar-width: 100px;
+          --progress-bar-height: 15px;
+        }
+
+        lablup-progress-bar.utilization {
+          --progress-bar-width: 80px;
+          margin-left: 10px;
+        }
+
+        lablup-shields {
+          margin: 1px;
+        }
+
+        mwc-icon {
+          --mdc-icon-size: 16px;
+        }
+
+        mwc-icon.schedulable {
+          --mdc-icon-size: 24px;
+        }
+
+        vaadin-grid {
+          border: 0;
+          font-size: 14px;
+        }
+    `];
+  }
+
+  firstUpdated() {
+    this.list_status = this.shadowRoot.querySelector('#list-status');
+    this.notification = globalThis.lablupNotification;
+    this.agentDetailDialog = this.shadowRoot.querySelector('#agent-detail');
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+  }
+
+  /**
+  * Change state to 'ALIVE' when backend.ai client connected.
+  *
+  * @param {Boolean} active - The component will work if active is true.
+  */
+  async _viewStateChanged(active: boolean) {
+    await this.updateComplete;
+    if (active === false) {
+      return;
+    }
+    // If disconnected
+    if (typeof globalThis.backendaiclient === 'undefined' || globalThis.backendaiclient === null || globalThis.backendaiclient.ready === false) {
+      document.addEventListener('backend-ai-connected', () => {
+        this.enableAgentSchedulable = globalThis.backendaiclient.supports('schedulable');
+        const status = 'ALIVE';
+        this._loadAgent(status);
+      }, true);
+    } else { // already connected
+      this.enableAgentSchedulable = globalThis.backendaiclient.supports('schedulable');
+      const status = 'ALIVE';
+      this._loadAgent(status);
+    }
+  }
+
+  /**
+  * Load an agents list with agent's backend.ai information.
+  *
+  * @param {string} status - The agent's backend.ai client status.
+  */
+  _loadAgent(status = 'running') {
+    if (this.active !== true) {
+      return;
+    }
+    this.list_condition = 'loading';
+    this.list_status.show();
+    const fields = ['id', 'status', 'available_slots', 'occupied_slots', 'architecture'];
+    switch (this.condition) {
+    case 'running':
```

Comment:
I think there are only two kinds of status settings in here,
If the condition is `terminated` then status will be set `TERMINATED`, in the other cases, it goes to `ALIVE`.
In that case, we shall change the code in a neater way like this:
```
switch (this.condition) {
  case 'running':
  case 'archived':
  default:
    status = 'ALIVE';
  case 'terminated':
    status =  'ALIVE';
}

Reply:
And also I wonder why we treat "archived" condition as `ALIVE`.
Do you know any clue for that @Sujin-Kim1 ?

Reply:
Then, the syntax of the switch can be changed as follows.
`status = this.condition === 'running' ? 'ALIVE' : 'TERMINATED';`

Reply:
I think referencing from model schema of `Agents` would be the best way.
https://github.com/lablup/backend.ai/blob/14373e7b9ffd70bcc2e8ccbeb6855636a6bb2bca/src/ai/backend/manager/models/agent.py#L53

According to the AgentStatus schema definition in the code above, It has four statuses. 
`ALIVE`, `LOST`, `TERMINATED`, `RESTARTING`.
I wonder whether we should cover `LOST` or `RESTARTING` here.

And there's one more thing that seems a bit awkward in this function `_loadAgentList` (perhaps because of the legacy... I guess). It receives status as an argument, but It manipulates by the `condition`.
Therefore I don't know the exact purpose of property how condition and status... maybe we should treat it as one unless there's certain reason to add "condition". 
I know in `backend-ai-agent-view`, there are two tabs for `backend-ai-agent-list` split by conditions, but we could give a better approach without using two purpose-overlapping property in vague.

Reply:
I just remove the status parameter in the `_loadAgentList` in both agent and agent-summary.
And how about handling the `LOST` and `RESTARTING` agents list when it is needed?